features/admin.branding.v1/components/custom-page-editor-page-layout.tsx

```diff
@@ -0,0 +1,175 @@
+     * The test id for the component.
+     */
+    "data-testid"?: string;
+    /**
+     * Custom Layout Mode.
+     */
+    customLayoutMode?: boolean;
+    /**
+     * Set custom layout mode.
+     */
+    setCustomLayoutMode?: (value: boolean) => void;
+}
+
+interface RouteParams {
+    templateTypeId: string;
+    templateId: string;
+}
+
+interface UpdatedContent {
+    html: string;
+    css: string;
+    js: string;
+}
+
+interface CustomPageEditorPageLayoutInterface extends IdentifiableComponentInterface{
+    /**
+     * Is the form is loading.
+     */
+    isLoading: boolean;
+    /**
+     * Is the form in a submitting state.
+     */
+    isSubmitting: boolean;
+    /**
+     * Is the form read only.
+     */
+    readOnly: boolean;
+}
+
+const CustomPageEditorPageLayout: FunctionComponent<CustomPageEditorPageLayoutPropsInterface> = (
+    props: CustomPageEditorPageLayoutPropsInterface &
+    CustomPageEditorPageLayoutInterface & RouteComponentProps<RouteParams>
+): ReactElement => {
+
+    const {
+        [ "data-testid" ]: testId,
+        customLayoutMode,
+        setCustomLayoutMode
+    } = props;
+
+    const handleBackButtonClick = (): void => {
+        setCustomLayoutMode(!customLayoutMode);
+    };
+
+    const { preference: brandingPreference } = useContext(BrandingPreferenceContext);
+    const customContent: BrandingPreferenceCustomContentInterface =
+        brandingPreference?.preference?.customContent?? {};
+
+    const [ html, setHtml ] = useState("");
+    const [ css, setCss ] = useState("");
+    const [ js, setJs ] = useState("");
+
+    useEffect(() => {
+        setHtml(customContent.htmlContent ?? "");
+        setCss(customContent.cssContent ?? "");
+        setJs(customContent.jsContent ?? "");
+    }, [ customContent ]);
+
+    const formRef: MutableRefObject<FormPropsInterface> = useRef<FormPropsInterface>(null);
+
+    const { updateBrandingCustomContent } = useContext(BrandingPreferenceContext);
+
+    const handleSaveAndPublish = (): void => {
+        updateBrandingCustomContent({
+            cssContent: css,
+            htmlContent: html,
+            jsContent: js
+        });
+    };
+
+    return (
+        <div>
+            <div
+                className = "back-button"
+                data-testid = "user-mgt-edit-user-back-button"
+                onClick = { handleBackButtonClick }
+                style = { {
+                    cursor: "pointer"
+                } }
+            >
+                <i aria-hidden="true" className="arrow left icon"></i>
+                    Go back
+            </div>
+            <div style = { { position: "relative" } }>
+                <Segment style = { { paddingBottom: "0", paddingLeft: "0", paddingRight: "0" } }>
+                    <EditorViewTabs
+                        html = { html }
+                        css = { css }
+                        js = { js }
+                        onContentUpdate = { ( updated: UpdatedContent ) => {
```

Comment:
Let's fix formatting. Please check and update all places.

```suggestion
                        html={ html }
                        css={ css }
                        js={ js }
                        onContentUpdate={ ( updated: UpdatedContent ) => {
```